css_parser.gemspec

```diff
@@ -1,23 +1,33 @@
-name = "css_parser"
+# frozen_string_literal: true
+
+name = 'css_parser'
 require "./lib/#{name}/version"
 
 Gem::Specification.new name, CssParser::VERSION do |s|
-  s.summary  = "Ruby CSS parser."
-  s.description  = "A set of classes for parsing CSS in Ruby."
-  s.email    = "code@dunae.ca"
+  s.summary = 'Ruby CSS parser.'
+  s.description = 'A set of classes for parsing CSS in Ruby.'
+  s.email    = 'code@dunae.ca'
   s.homepage = "https://github.com/premailer/#{name}"
-  s.author  = "Alex Dunae"
-  s.files = Dir.glob("lib/**/*") + ["MIT-LICENSE"]
-  s.license = "MIT"
+  s.author = 'Alex Dunae'
+  s.files = Dir.glob('lib/**/*') + ['MIT-LICENSE']
+  s.license = 'MIT'
+  s.required_ruby_version = '>= 2.2' # rubocop:disable Gemspec/RequiredRubyVersion
 
-  s.metadata['changelog_uri'] = "https://github.com/premailer/css_parser/blob/master/CHANGELOG.md"
-  s.metadata['source_code_uri'] = "https://github.com/premailer/css_parser"
-  s.metadata['bug_tracker_uri'] = "https://github.com/premailer/css_parser/issues"
+  s.metadata['changelog_uri'] = 'https://github.com/premailer/css_parser/blob/master/CHANGELOG.md'
+  s.metadata['source_code_uri'] = 'https://github.com/premailer/css_parser'
+  s.metadata['bug_tracker_uri'] = 'https://github.com/premailer/css_parser/issues'
 
   s.add_runtime_dependency 'addressable'
 
   s.add_development_dependency 'bump'
   s.add_development_dependency 'maxitest'
   s.add_development_dependency 'rake'
   s.add_development_dependency 'webrick'
+
+  # We're using newest rubocop to get newest cops, but it's not
+  # compatible with older rubies
+  if Gem::Version.new(RUBY_VERSION) > Gem::Version.new('2.4')
```

Comment:
it should work on 2.4 ... and if not just drop 2.4 compatability and bump the target ruby version to 2.5 ?

Reply:
It's doesn't work on 2.2-2.3, which are on CI. We don't really need anything from newer syntax (I made the changes on 2.7 and then `git rebase -i upstream/master -x `bundle exec rake'` on 2.2 & 3.0, tests passed just fine.
So it's not really hurts anyone except strange people that using < 2.4 for development and will have rubocop failures only on CI.

Speaking about that: should I add rubocop to `the default rake task?

Reply:
yeah sounds good, but don't run it on ci for each ruby version

Reply:
we can't have rubucop target a newer version then our minimum version ... so prefer to drop EOL rubies